starter-core/src/main/java/io/micronaut/starter/feature/awslambdacustomruntime/AwsLambdaCustomRuntime.java

```diff
@@ -64,8 +64,8 @@ public void processSelectedFeatures(FeatureContext featureContext) {
         if (awsLambda.supports(featureContext.getApplicationType()) && !featureContext.isPresent(AwsLambda.class)) {
             featureContext.addFeature(awsLambda);
         }
-        if (!featureContext.isPresent(HttpClient.class)) {
-            featureContext.addFeature(httpClient);
+        if (!featureContext.isPresent(HttpClientJdk.class)) {
```

Comment:
this should be: 

```suggestion
        if (!featureContext.isPresent(HttpClientFeature.class)) {
```?

Reply:
resolved via https://github.com/micronaut-projects/micronaut-starter/pull/2272/commits/cfd040e1ab589e6c8431163a4300e1b355768f3c